### DISCUSSION TITLE
Goals

### DISCUSSION THREAD

**Discussion by cirrusflyer** [umami-software/umami#1815] [Ideas]
Can Umami do something like this?:

https://plausible.io/docs/pageview-goals

**Comment by mikecao** [comment]
I think we can do it using localStorage so we wouldn't need to touch the database. I'm trying to understand what the use case would be though. You can just want a goal count of views and events? Do the goals go away after some time?

**Comment by cirrusflyer** [comment]
Thanks Mike.  We just use for a quick way to get stats on a particular page or set of pages, that might not be in the top counted pages.  I realize this could probably be done using your "Track events" method, but this feature doesn't require any coding, so any user could create these.  Yes, just want a goal count of views and events.  The goals remain as long as we don't remove them.
